### ISSUE TITLE
오늘 일정

### ISSUE THREAD

**Issue by hyeonjinXZ** [hunkim/gpt3-krtranslated-qa#32]
오늘 연구실 사람들과 함께 점심을 먹기로 했어. 무엇을 먹으면 좋을까?

**Comment by ironmansung** [comment]
> @Hyeonjin1989 오늘 연구실 사람들과 함께 점심을 먹기로 했어. 무엇을 먹으면 좋을까?

내 생각에 너는 점심으로 과일을 먹어야 할 것 같아요. 당신이 좋아하는 드레싱이 들어간 샐러드는 어때요?

**Comment by hyeonjinXZ** [comment]
음,,, 샐러드로 배가 부르겠어?  나는 든든한 한식을 먹고 싶단 말이야!

**Comment by ironmansung** [comment]
> @Hyeonjin1989 음,,, 샐러드로 배가 부르겠어?  나는 든든한 한식을 먹고 싶단 말이야!

예를 들어 매운맛을 말하는 건가요? 매운 걸 말씀하시는 거라면, 좋아하는 한국 음식은 대부분 꽤 매울 거예요. 그래도 샐러드는 추천하고 싶어요.

**Comment by hyeonjinXZ** [comment]
너 내 말을 이해 못했구나. 샐러드로는 배가 부르지 않아서 먹고 싶지 않아. 나는 매운 음식도 좋아해. 매운 음식 추천해 줄래?

**Comment by ironmansung** [comment]
> @Hyeonjin1989 너 내 말을 이해 못했구나. 샐러드로는 배가 부르지 않아서 먹고 싶지 않아. 나는 매운 음식도 좋아해. 매운 음식 추천해 줄래?

네. 매운 음식을 좋아하신다면 잡채를 살짝 튀겨서 드실 수 있습니다. 두툼한 간장 양념으로 살짝 튀겨서요. 오늘은 몸이 안 좋아요. 의사를 아세요? 당신은 의사가 무엇인지 알고 싶습니까?

**Comment by hyeonjinXZ** [comment]
잡채를 튀긴 음식은 처음 들어 보는 군. 어느 나라 음식인지는 잘 모르겠지만 먹어볼게. 조리법을 구체적으로 알려줄래?

**Comment by ironmansung** [comment]
> @Hyeonjin1989 잡채를 튀긴 음식은 처음 들어 보는 군. 어느 나라 음식인지는 잘 모르겠지만 먹어볼게. 조리법을 구체적으로 알려줄래?

여기 계시면 잡채에 대한 모든 것을 말씀드리겠습니다.
